.gitignore

```diff
@@ -155,3 +155,9 @@ yarn-debug.log*
 yarn-error.log*
 
 backend/media/*
+
+backend/CompDatabase/comps.db
+backend/Report.pdf
```

Comment:
the backend/Report.pdf can be deleted since the path was changed in the pdf generation file